config.go

```diff
@@ -18,6 +18,7 @@ var (
 	defaultConfigRoot         = "/etc/grim"
 	defaultResultRoot         = "/var/log/grim"
 	defaultWorkspaceRoot      = "/var/tmp/grim"
+	defaultTimeOutSeconds	  = 5
```

Comment:
Seems aggressive.  What do you think @bhand-mm?  I'm thinking 30 seconds is probably better.